apps/src/code-studio/progressRedux.ts

```diff
@@ -87,11 +87,15 @@ export interface ProgressState {
   unitCompleted: boolean | undefined;
 }
 
-export interface MilestoneReport {
+export interface MilestoneReport extends OptionalMilestoneData {
   app: string;
   result: boolean;
   testResult: number;
+}
+
+interface OptionalMilestoneData {
   program?: string;
+  submitted?: string;
```

Comment:
Why a string and not a boolean?

Reply:
That's what the server expects 🤷‍♀️ it didn't work when I sent a boolean. I can add a comment explaining this.